recipes/r-pegboard/meta.yaml

```
@@ -0,0 +1,77 @@
+{% set version = '0.7.9' %}
+{% set posix = 'm2-' if win else '' %}
+
+package:
+  name: r-pegboard
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url: https://github.com/carpentries/pegboard/archive/refs/tags/{{ version }}.tar.gz
+  sha256: ee9c429e792817ea269b43c98bc95bcab62aaff41d30754377cd4d7e73539611
+
+build:
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  missing_dso_whitelist:
+    - '*/R.dll'        # [win]
+    - '*/Rblas.dll'    # [win]
+    - '*/Rlapack.dll'  # [win]
+
+requirements:
+  build:
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}git
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-r6
+    - r-commonmark
+    - r-fs >=1.5.0
+    - r-glue
+    - r-purrr
+    - r-tinkr >=0.2.0
+    - r-xml2
+    - r-xslt
+    - r-yaml
+  run:
+    - r-base
+    - r-r6
+    - r-commonmark
+    - r-fs >=1.5.0
+    - r-glue
+    - r-purrr
+    - r-tinkr >=0.2.0
+    - r-xml2
+    - r-xslt
+    - r-yaml
+
+test:
+  commands:
+    - $R -e "library('pegboard')"           # [not win]
+    - "\"%R%\" -e \"library('pegboard')\""  # [win]
```

Since there was an issue with `r-tinkr`, this could be a good place to run unit tests to check for functionality.

```suggestion
  requires:
    - r-testthat
  source_files:
    - tests/
  commands:
    - $R -e "library('pegboard')"                                                      # [not win]
    - $R -e "testthat::test_file('tests/testthat.R', stop_on_failure=TRUE)"           # [not win]
    - "\"%R%\" -e \"library('pegboard')\""                                             # [win]
    - "\"%R%\" -e \"testthat::test_file('tests/testthat.R', stop_on_failure=TRUE)\""  # [win]
```